features_and_tools/multi-tenancy.rst

```diff
@@ -14,9 +14,10 @@ Firely Server offers *virtual* multi-tenancy. All the resources are still stored
 The basics are simple:
 
 - On every request, the tenant is read from either a specific HTTP header or a claim in the authorization token.
+- If no tenant is specified, depending whether your operation configuration requires it, the special `shared` tenant might be used internally.
 - If the request creates or updates any resource, the tenant is applied to each of those resources. This is done by adding a *security label* to the ``meta.security`` element of the resource.
 - For both reading and writing resources, the tenant is used to limit access to only the resources having a matching security label.
-- Upon reading, the tenant security label is removed from the resource and thus not visible from the outside.
+- Upon reading, the tenant security label is (by default) removed from the resource and thus not visible from the outside.
```

Comment:
Avoid parentheses in text, they make text unclear. It is either by default or not. In this case, it is.
```suggestion
- Upon reading, by default, the tenant security label is removed from the resource and thus not visible from the outside. If you want to communicate the label to the client, set ``RemoveTenantLabelFromResponse`` to ``false`` in the settings.
```